File: Java/session_03/src/p1/Student.java
package p1;

public class Student {
	public static int fees = 1000;

	Student() {
		fees = fees + 10;
	}

//	static {
//		fees = 400;
//	}

//	static {
//		fees += 500;
//	}

	public static void main(String[] args) {
		System.out.println(Student.fees);
//		System.out.println("Fees: " + fees);
	}
}

File: C# .Net/Projects/LinqExample/Model/Student.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace LinqExample.Model;

public class Student
{
    [Key] public int StudentId { get; set; }

    [Required(ErrorMessage = "Student name is required")]
    [StringLength(100, ErrorMessage = "Student name should be at leat 100")]
    public string StudentName { get; set; } = string.Empty;

    [Range(15, 100, ErrorMessage = "The age must be in the range!")]
    public int Age { get; set; }

    [ForeignKey("Department")] public int DeptId { get; set; }
    
    public Department Department { get; set; }
}